tools/bakeddata-scripts/src/main.rs

```diff
@@ -29,7 +29,7 @@ const COMPONENTS: &[(&str, &[DataKey], &str)] = &[
     ("timezone", icu::timezone::provider::KEYS, REPO_VERSION),
     (
         "experimental",
```

Comment:
Suggestion: do you want to make `icu_experimental` be `icu::experimental` in baked data metacrate mode?

Reply:
I guess yes